js/ui/select_box.js

```diff
@@ -383,15 +383,15 @@ var SelectBox = DropDownList.inherit({
     },
 
     _renderInputValue: function() {
-        this._renderInputValueAsync = function() {
```

Comment:
This method will be assigned to $.noop on dispose. It is the reason why this method should be assigned dynamically